OracleWebCenterContent/dockerfiles/README.md

```diff
@@ -0,0 +1,274 @@
+
+```
+sudo mkdir -p /<YOUR_HOST_DIRECTTORY_PATH>/wccontent
+sudo chown 1000:1000 /<YOUR_HOST_DIRECTTORY_PATH>/wccontent
+```
+
+All container operations are performed as **'oracle'** user.
+
+**Note**: If a user already exists with **'-u 1000 -g 1000'** then use the same user. Or modify any existing user to have uid-gid as **'-u 1000 -g 1000'**
+
+## 3.5. Database
+You need to have a running database container or a database running on any machine. 
+The database connection details are required for creating WebCenter Content specific RCU schemas while configuring WebCenter Content domain. 
+
+The Oracle Database image can be pulled from [Oracle Container Registry](https://container-registry.oracle.com) or build your own using the [Oracle Database Dockerfiles and scripts]((https://github.com/oracle/docker-images/tree/master/OracleDatabase)) in this repo.
+
+## 3.6. Docker Security Configuration
+
+For detailed instructions of security best practices, please refer to this [documenation](https://docs.oracle.com/en/operating-systems/oracle-linux/docker/docker-security.html#docker-security-components).
+
+# 4. Building Oracle WebCenter Content Image
+
+To build/use a WebCenter Content image you should have Oracle FMW Infrastrucure image.
+
+## 4.1. Pulling Oracle FMWInfrastructure Install Image
+
+Get Oracle FMWInfrastructure Image -
+
+> 1. Sign in to Oracle Container Registry. Click the Sign in link which is on the top-right of the Web page.
+> 2. Click Middleware and then click Continue for the fmw-infrastructure repository.
+> 3. Click Accept to accept the license agreement.
+> 4. Use following commands to pull Oracle Fusion Middleware infrastructure base image from repository :
+Refer Documentation here: https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure
+```
+docker login container-registry.oracle.com
+docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+docker tag  container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4 oracle/fmw-infrastructure:12.2.1.4.0
+docker rmi container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+
+```
+Alternatively to build this image yourself, please refer this [link](https://github.com/oracle/docker-images/blob/master/OracleFMWInfrastructure/README.md).
+
+## 4.2. Building Docker Image for WebCenter Content
+
+You have to download the binary for WebCenter Content shiphome and put it in place. The binaries can be downloaded from the [Oracle Software Delivery Cloud](https://edelivery.oracle.com/). Search for "Oracle WebCenter Content" and download the version which is required.
+Extract the downloaded zip files and copy `fmw_12.2.1.4.0_wccontent.jar` file under `dockerfiles/12.2.1.4` .
+Checksum of shiphome binary needs to be mentioned in this [file](12.2.1.4/Checksum). Set the Proxies in the environment before building the image as required, go to directory located at OracleWebCenterContent/dockerfiles/ and run these commands -
```

Comment:
I'm confused about this line. Generating the checksum on the client defeats the purpose of the checksum, which is to ensure the file downloaded correctly. _You_ must provide the checksum for each file required by your build process so that when the checksum is verified, we are sure the package is valid and correct.

You need to wrap OracleWebCenterContent/dockerfiles/  with `` ` `` characters, i.e. `` `OracleWebCenterContent/dockerfiles/ ` `` so that it is rendered in a monospace font.

Please change "proxies" to lower case as well.